clients/venice-admin-tool/src/main/java/com/linkedin/venice/KafkaTopicDumper.java

```diff
@@ -363,58 +363,48 @@ String getChunkMetadataLog(PubSubMessage<KafkaKey, KafkaMessageEnvelope, Long> r
       }
 
       final ChunkKeyValueTransformer.KeyType keyType = ChunkKeyValueTransformer.getKeyType(messageType, schemaId);
-      ChunkId firstChunkId = getFirstChunkId(keyType, kafkaKey, kafkaMessageEnvelope);
-
-      if (firstChunkId == null) {
-        String chunkMetadataFormatWithoutFirstChunkMd = " ChunkMd=(type:%s)";
-        return String.format(chunkMetadataFormatWithoutFirstChunkMd, keyType);
-      } else {
-        String chunkMetadataFormatWithFirstChunkMd = " ChunkMd=(type:%s, FirstChunkMd=(guid:%s,seg:%d,seq:%d))";
-        return String.format(
-            chunkMetadataFormatWithFirstChunkMd,
-            keyType,
-            GuidUtils.getHexFromGuid(firstChunkId.producerGUID),
-            firstChunkId.segmentNumber,
-            firstChunkId.messageSequenceNumber);
+      switch (keyType) {
```

Comment:
Why move this from a function?

Reply:
No strong reason, but I realized that the chunk id is not always a `firstChunkId`. Before this change, we weren't logging the chunk index.

If I still keep the function, the name would have been something like `getChunkId` - but I would still have to check outside the function if it was CVM - and then not print the chunk index

Reply:
This is the output now:
```
REG DELETE Offset:579726 ProducerMd=(guid:<guid1>,seg:14,seq:46,mts:1692641292827,lts:-2) LeaderMd=(host:<host1>,uo:11976907,ukcId:1) ChunkMd=(type:WITH_VALUE_CHUNK, ChunkIndex: 0, FirstChunkMd=(guid:<guid0>,seg:0,seq:202995))
REG DELETE Offset:579727 ProducerMd=(guid:<guid1>,seg:14,seq:47,mts:1692641292827,lts:-2) LeaderMd=(host:<host1>,uo:11976907,ukcId:1) ChunkMd=(type:WITH_VALUE_CHUNK, ChunkIndex: 1, FirstChunkMd=(guid:<guid0>,seg:0,seq:202995))
REG PUT Offset:579728 ProducerMd=(guid:<guid1>,seg:14,seq:48,mts:1692641292836,lts:-1) LeaderMd=(host:<host1>,uo:-1,ukcId:-1) ChunkMd=(type:WITH_VALUE_CHUNK, ChunkIndex: 0, FirstChunkMd=(guid:<guid1>,seg:14,seq:48))
REG PUT Offset:579729 ProducerMd=(guid:<guid1>,seg:14,seq:49,mts:1692641292855,lts:-1) LeaderMd=(host:<host1>,uo:-1,ukcId:-1) ChunkMd=(type:WITH_VALUE_CHUNK, ChunkIndex: 1, FirstChunkMd=(guid:<guid1>,seg:14,seq:48))
REG PUT Offset:579730 ProducerMd=(guid:<guid1>,seg:14,seq:50,mts:1692641292859,lts:-2) LeaderMd=(host:<host1>,uo:11976908,ukcId:1) ChunkMd=(type:WITH_CHUNK_MANIFEST, FirstChunkMd=(guid:<guid1>,seg:14,seq:48))
REG DELETE Offset:579731 ProducerMd=(guid:<guid1>,seg:14,seq:51,mts:1692641292859,lts:-2) LeaderMd=(host:<host1>,uo:11976908,ukcId:1) ChunkMd=(type:WITH_VALUE_CHUNK, ChunkIndex: 0, FirstChunkMd=(guid:<guid1>,seg:14,seq:43))
```